### DISCUSSION TITLE
friends/

### DISCUSSION THREAD

**Discussion by giscus** [waynerv/waynerv.github.io#36] [Announcements]
# friends/

Spaceack's Blog 喜提第一位博友 ！ 郭帅的个人博客 博主是浙江大学计算机在读研究生，关注 C++、Linux 和 Python 等技术领域。 🌻长街短梦 一个喜欢折腾博……

https://waynerv.com/friends/

**Comment by zhaoqingqing** [comment]
https://www.cnblogs.com/zhaoqingqing/
你好，我是赵青青   一名在【网易游戏】做游戏开发的程序员，擅长Unity3D，自研引擎，游戏开发，.NET等领域。

**Comment by qixing-jk** [comment]
https://blog.qixing1217.top/
您好，我是七行，一个喜欢探索各种新玩意的冲浪爱好者🏄‍♂️
**博客主要内容：** 

- 亲身使用过的好用的工具推荐
- 使用遇到的各种问题的解决方案
